doc/Makefile

```diff
@@ -20,6 +20,7 @@ CP            := cp
 ROBOT_XUNIT   = $(ROBOTDIR)/report.xml
 ITEST_PLAN    = $(OUTDIR)/itest_plan.rst
 LOG_FILE      = log.html
+LAYER         ?= FLASH
```

Comment:
I would leave empty to start with and then in C run twice - once with assigned and once with not?

Reply:
2a6b733738561d1b58eaf544014dd27ac7f27734 and 25024f143966bcadcc2e5ac66609cd143d5a1390